### PR TITLE
Updating footer

### PR DESCRIPTION
**Pull request by briwylde08**
Change developers-community and add community links
All tools to link to dev docs tools page 
Remove get started from resources

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -136,23 +128,24 @@ const config = {
...
+          title: "Community",
           items: [
             {
-              label: "Developer Resources",
-              href: "https://www.stellar.org/developers-legacy",
+              label: "Stellar Developer Discord",
+              href: "https://discord.com/invite/zVYdY3ktTn",
             },
             {
-              label: "Status",
```
Comment (tyvdh) [inline docusaurus.config.js:150]: We should definitely keep this link. It's one of the most important for pro tier devs. We should also include this one https://dashboard.stellar.org/

File: docusaurus.config.js [hunk 2]
```diff
@@ -136,23 +128,24 @@ const config = {
...
         },
         {
-          title: "Developers",
+          title: "Community",
           items: [
             {
-              label: "Developer Resources",
-              href: "https://www.stellar.org/developers-legacy",
+              label: "Stellar Developer Discord",
+              href: "https://discord.com/invite/zVYdY3ktTn",
```
Comment (tyvdh) [inline docusaurus.config.js:140]: This is a non-expiring invite yes?

File: docusaurus.config.js (hunk 1 above)
Comment (tyvdh) [reply docusaurus.config.js:150]: Both of these links likely belong in tools

File: docusaurus.config.js [hunk 3]
```diff
@@ -100,11 +100,7 @@ const config = {
           title: "Resources",
           items: [
             {
-              label: "Get Started",
-              href: "https://www.stellar.org/start",
-            },
-            {
-              label: "Learn",
+              label: "Stellar Learn",
```
Comment (tyvdh) [inline docusaurus.config.js:103]: Probably don't need this? We should link to the Stellar homepage somewhere but probably in the header not the footer

**Review by tyvdh — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (briwylde08) [reply docusaurus.config.js:150]: I was just trying to shorten the list so the footer isn't ridiculously long. Just want to keep the top 4 most used then link to the rest of the tools. Most used are? Account Viewer, Dashboard, Ledger Explorer, Status? Or something else?

**Pull request merged**